packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -804,9 +806,27 @@ export default class Meetings extends WebexPlugin {
       );
 
       this.meetingCollection.set(meeting);
+      const isMeetingActive = destination.fullState.active;
+      const handleLocusMercuryIsSource = (source === 'handleLocusMercury');
 
       try {
-        await meeting.fetchMeetingInfo({});
+        // if no participant has joined the scheduled meeting (meaning meeting is not active) and we still get a locusEvent,
+        // it means the meeting will start in 5-6 min. In that case, we want to fetchMeetingInfo
+        // between 5 and 2 min (random between 3 minutes) before the meeting starts
+        // to avoid a spike in traffic to the wbxappi service
+        const {startTime} = destination.meeting;
```

Comment:
Have we tried to see what happens when its a 1:1 call or adhoc meeting do they have startTime all the time